### PR TITLE
add template content for icons, typography, lists, tables

### PR DESCRIPTION
**Pull request by hippogriffic**
ignore the compiled templates / css we have to commit this for gh pages

@incuna/frontend please review

image

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -64,12 +64,17 @@ module.exports = function (grunt) {
                         color8: '#ab7e5b',
                         color9: '#5b755d'
                     },
+                    icons: {
+                        alert: 'alert',
+                        flag: 'flag',
+                        blah: 'blah'
```
Comment (maciejpi) [inline Gruntfile.js:70]: blah?

Comment (hippogriffic) [inline Gruntfile.js:70]: just added these to generate some some html for styling, they will be remove when I get around to build a proper demo version

File: index.html [hunk 2]
```diff
@@ -255,43 +255,304 @@ <h2 class="section-title">Components</h2>
...
                 
-                    <div class="icons-section" id="icons-example">
-                        <h2>Icons</h2>
+                    <div class="section icons-section" id="icons-example">
+                        <h2 class="section-title">Icons</h2>
                         
+
+    <h3 class="section-sub-title">Icon usage</h3>
+
+
```
Comment (maciejpi) [inline index.html:264]: so is this compiled? is that the reason for the gaps?

Comment (wytrych) [inline index.html:264]: If it's compiled maybe there's no point in commiting this file?

**Comment by maciejpi** [comment]
@hippogriffic tests failing

File: index.html (hunk 2 above)
Comment (hippogriffic) [inline index.html:264]: yeah it's compiled, you can tell because it's ugly. And unfortunately we have to commit it for the github page to work

**Comment by hippogriffic** [comment]
yeah that's @ninjanails 's fault :P

File: index.html (hunk 2 above)
Comment (henrahmagix) [inline index.html:264]: @hippogriffic You can add it to `.gitignore` so it's ignored, then merge into `gh-pages` and remove from `.gitignore`, then commit it there. That means you can compile in `master` and not have to commit it, but then when merging into `gh-pages` you compile it and commit.

Comment (henrahmagix) [inline index.html:264]: You can add a script to run something like this:

``` shell
git checkout gh-pages
git pull
git merge origin/master
grunt build
git commit -a 'Update'
git push
```

**Comment by ninjanails** [comment]
Tests are working :D

**Pull request merged**